opentelemetry-sdk/src/logs/record.rs

```diff
@@ -254,7 +258,11 @@ mod tests {
             severity_text: Some(Cow::Borrowed("ERROR")),
             severity_number: Some(Severity::Error),
             body: Some(AnyValue::String("Test body".into())),
-            attributes: Some(vec![(Key::new("key"), AnyValue::String("value".into()))]),
+            attributes: {
```

Comment:
Can we modify the test to do what end users will do to populate attributes, instead of relying on internal impln?

Reply:
This is supposed to be unit-test to validate internal constructs. The user can never create the log record by direct assignment as tested here. And the test won't need any changes in future if we change the attributes implementation details.